widgets/htdocs/components/95_PDB.js

```diff
@@ -0,0 +1,1979 @@
+/*
+ * Copyright [1999-2015] Wellcome Trust Sanger Institute and the EMBL-European Bioinformatics Institute
+ * Copyright [2016-2017] EMBL-European Bioinformatics Institute
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *      http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+Ensembl.Panel.PDB = Ensembl.Panel.Content.extend({
+  init: function() {
+    var panel = this;
+    this.base.apply(this, arguments);
+
+    // Load a specific plugin to highlight the different parts of the protein with different colour
+    // and the focus variant with its corresponding amino acid(s) side chain(s)
+    $.getScript('/pdbe/litemol-custom-theme.js');
+
+    // Setup variables
+    this.species = 'Homo_sapiens';    
+
+
+    // Check if we can get the REST URL from the webiste
+    this.rest_url_root         = this.params['ensembl_rest_url'];
+    this.rest_var_url          = this.rest_url_root+'/variation/'+this.species+'/';
+    this.rest_overlap_url      = this.rest_url_root+'/overlap/region/'+this.species+'/';
+    this.rest_pr_url           = this.rest_url_root+'/overlap/translation/';
+    this.rest_lookup_url       = this.rest_url_root+'/lookup/id/';
+    this.pdbe_url_root         = 'https://www.ebi.ac.uk/pdbe';
```

Comment:
This needs to be retrieved from SiteDefs same as you did for rest_url_root

Reply:
Do you mean for the `pdbe_url_root` ?

Reply:
yes, sorry it wasnt clear, still getting used to comment on PR.